### PR TITLE
Address PR comments.

### PR DESCRIPTION
**Pull request by issackjohn**

### CODE REVIEW

File: resources/tentative/complex-static-html/src/big-dom-generator/gen-css.js [hunk 1]
```diff
@@ -125,32 +130,26 @@ const randomWeighted = (options, probabilities) => {
 
 const buildSelectors = (element, depth, oldCombinator, selectorLength, maxSelectorLength, isMatching) => {
     // if nonMatching we add a view-<random-index> class selector that is guaranteed to not have targeted children in the todoMVC
-    if ((!isMatching && !depth) || selectorLength >= maxSelectorLength || !element)
+    if (depth < 3 || selectorLength >= maxSelectorLength)
```
Comment (issackjohn) [inline resources/tentative/complex-static-html/src/big-dom-generator/gen-css.js]: I prefer `depth < 3` for simplicity but I think 
`!element || element.tagName === "HTML" || element.tagName === "BODY"`
is less prone to errors in depth calculations from `getNextDepth(combinator, depth)`?

WDYT? @lpardosixtosMs

**Comment by issackjohn** [comment]
There's no generated rules for `show-more`, `ribbon` and `tree-area` but there are rules for `main-ui`, `top-bar`, `todo-area`.

File: resources/tentative/complex-static-html/src/big-dom-generator/gen-css.js (hunk 1 above)
Comment (lpardosixtosMs) [reply resources/tentative/complex-static-html/src/big-dom-generator/gen-css.js]: I'd prefer if you compute the initial depth relative to the main-ui element, that way depth 0 is `.main-ui` and we don't have selectors involving the `body` tag

File: resources/tentative/complex-static-html/src/big-dom-generator/html-markup.js [hunk 2]
```diff
@@ -29,8 +29,8 @@ export const ANGULAR_TODO_MVC_HTML_MARKUP = `
 export const TODO_MVC_HTML_MARKUP = `
 <div class="main-ui">
     <div class="show-more"></div>
-    <div class="top-bar"/></div>
-    <div class="ribbon"/></div>
+    <div class="ribbon"></div>
```
Comment (lpardosixtosMs) [inline resources/tentative/complex-static-html/src/big-dom-generator/html-markup.js:32]: you can do `<div class="ribbon"/>` and remove the closing `</div>` tag.

Comment (issackjohn) [reply resources/tentative/complex-static-html/src/big-dom-generator/html-markup.js:32]: JSDOM doesn't handle that well. the resulting output is
```html
<div class="main-ui">
    <div class="show-more">
    <div class="ribbon">
    <div class="top-bar">
    <div class="tree-area">
    <div class="todo-area">
        <div class="todoholder">
            <section class="todoapp">
                <header class="header" data-testid="header"></header>
                <main class="main" data-testid="main">
                    <div class="toggle-all-container">
                        <input class="toggle-all" type="checkbox" data-testid="toggle-all">
                        <label class="toggle-all-label" for="toggle-all">Toggle All Input</label>
                    </div>
                    <ul class="todo-list" data-testid="todo-list"></ul>
                </main>
            </section>
        </div>
    </div>
</div>
</div></div></div></div>
```

Maybe it's a configuration issue. I'll take a quick look.

Comment (lpardosixtosMs) [reply resources/tentative/complex-static-html/src/big-dom-generator/html-markup.js:32]: Good to know, thanks!

**Review by lpardosixtosMs — APPROVED** [review]

**Pull request merged**